src/PlaywrightSharp.Chromium/ChromiumBrowserType.cs

```diff
@@ -202,7 +202,7 @@ public async Task<IBrowserApp> LaunchBrowserAppAsync(LaunchOptions options = nul
                     }
 
                     var transport = await BrowserHelper.CreateTransportAsync(browserApp.ConnectOptions).ConfigureAwait(false);
-                    await transport.SendAsync("Browser.close", new BrowserCloseRequest { Id = ChromiumConnection.BrowserCloseMessageId }).ConfigureAwait(false);
+                    await transport.SendAsync(new BrowserCloseRequest().Command).ConfigureAwait(false);

```

Comment:
Can't we make `Command` static? you could have one static and one instance for the interface.

Reply:
I'm pretty sure this is the only place we send a message directly to the transport abstraction. I think that only if we have another case like this then it might be a good enough excuse to have a static property